doc/particles.md

```diff
@@ -96,110 +99,110 @@ game.add(
+More examples of using these behaviors together are available [here](https://github.com/flame-engine/flame/blob/main/examples/lib/stories/utils/particles.dart). All the implementations are available in the [particles](https://github.com/flame-engine/flame/tree/main/packages/flame/lib/src/particles) folder on the Flame repository.
 
 ## Translated Particle
 
-Simply translates underlying `Particle` to a specified `Offset` within the rendering `Canvas`.
+Simply translates underlying `Particle` to a specified `Vector2` within the rendering `Canvas`.
 Does not change or alter its position, consider using `MovingParticle` or `AcceleratedParticle` where change of position is required.
 Same effect could be achieved by translating the `Canvas` layer.
 
 ```dart
 game.add(
     ParticleComponent(
         particle: TranslatedParticle(
-            // Will translate child Particle effect to
-            // the center of game canvas
-            offset: game.size.center(Offset.zero),
+            // Will translate child Particle effect to the center of game canvas.
+            offset: game.size / 2,
             child: Particle(),
-        )
-    )
+        ),
+    ),
 );
 ```
 
 ## Moving Particle
 
-Moves child `Particle` between `from` and `to` `Offset`s during its lifespan. Supports `Curve` via `CurvedParticle`.
+Moves child `Particle` between `from` and `to` `Vector2`s during its lifespan. Supports `Curve` via `CurvedParticle`.
 
 ```dart
 game.add(
     ParticleComponent(
         particle: MovingParticle(
-            // Will move from corner to corner of the
-            // game canvas
-            from: game.size.topLeft(Offset.zero),
-            to: game.size.bottomRight(Offset.zero),
+            // Will move from corner to corner of the game canvas.
+            from: Vector2.zero(),
+            to: game.size,
             child: Particle(),
-        )
-    )
+        ),
+    ),
 );
 ```
 
 ## Accelerated Particle
 
-A basic physics particle which allows you to specify its initial `position`, `speed` and `acceleration` and let the `update` cycle do the rest. All three are specified as `Offset`s, 
+A basic physics particle which allows you to specify its initial `position`, `speed` and `acceleration` and let the `update` cycle do the rest. All three are specified as `Vector2`s, 
 which you can think of as vectors. It works especially well for physics-based "bursts", but it is not limited to that.
-Unit of the `Offset` value is _logical px/s_. So a speed of `Offset(0, 100)` will move a child `Particle` by 100 logical pixels of the device every second of game time.
+Unit of the `Vector2` value is _logical px/s_. So a speed of `Vector2(0, 100)` will move a child `Particle` by 100 logical pixels of the device every second of game time.
```

Comment:
Speed is px/s, acceleration is px/s² and position is just what it is right?

Reply:
Not sure what you mean here :-)

Reply:
It is talking about three different values and they have different units, which is a bit unclear.

Reply:
Suggestion for rewrite?